Create rule S6414[terraform]: Excluding users or groups activities from audit logs is security-sensitive (#805)
SonarSource/rspec: rules/S6414/message.adoc

=== Message

> Make sure the audit logs member exemption is safe here.

Reviewer: ```suggestion
Make sure excluding members activity from audit logs is safe here.
```
